scripts/oc-clear-accounts.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Where this script is used?

Reply:
it isn't, it was part of an experiment I abandoned. I'll remove it.